sql/7_0_1-to-7_0_2_upgrade.sql

```diff
@@ -236,7 +236,7 @@ INSERT INTO `list_options` (`list_id`, `option_id`, `title`) VALUES ('lists', 'r
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`) VALUES ('recent_patient_columns', 'fname', 'First Name', '10');
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`) VALUES ('recent_patient_columns', 'mname', 'Middle Name', '20');
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`) VALUES ('recent_patient_columns', 'lname', 'Last Name', '30');
-INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`) VALUES ('recent_patient_columns', 'dob', 'Date of Birth', '40');
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`) VALUES ('recent_patient_columns', 'DOB', 'Date of Birth', '40');
```

Comment:
This fix should also be in patch.sql so will be fixed in first patch
if dob exist, then change to DOB (i am worried that we may need to add another #IfNotRow2D that is case sensititive maybe #IfNotRow2Dci which will use a binary character set in the underlying queries (can see examples of this in the authentication/login queries when finding usernames)

Reply:
fun fun :)